Configure iOS setup. Add app Icon

Knyazev-AI opened the pull request:
1. Добавлен значок приложения в ресурсы проекта;
2. Установлена минимальная версия iOS 13.0;
3. Установлена портретная ориентации приложения;
4. Установлена поддержка операционной системы - только iOS;
5. Через SPM установлены точные версии библиотек: KeychainAccess, GoogleSignIn, SnapKit
6. Проект протестирован на устройствах:
iPhone 7
iPhone 7 Plus
iPhone 11
iPhone 13 mini
iPhone 14 Pro Max
iPhone 15
iPhone SE(1-е поколение)

Commit:
Add: application icon to project resources

Commit:
Configure: iOS version, orientation, device type, libraries

The minimum iOS version that supports the app is 13.0

Application orientation support is installed - portrait

Operating system support is installed - iOS only

The exact versions of the libraries are installed through Swift Package Manager:
- KeychainAccess version 4.2.2
- GoogleSignIn-iOS version 7.1.0
- SnapKit version 5.7.1

The project has been tested on devices:
- iPhone 7
- iPhone 7 Plus
- iPhone 11
- iPhone 13 mini
- iPhone 14 Pro Max
- iPhone 15
- iPhone SE(3rd generation)

Commit:
Merge branch 'feature/ios-setup-project' into feature/ios-setup-project-stack-compliance

Curshet commented on MyBook.xcodeproj/project.pbxproj:
```diff
@@ -12,6 +12,9 @@
 		7523ECA82BA6AE1200707DB7 /* InfoPlist.strings in Resources */ = {isa = PBXBuildFile; fileRef = 7523ECAA2BA6AE1200707DB7 /* InfoPlist.strings */; };
+		755C9F5F2BC42B9E00E1F089 /* KeychainAccess in Frameworks */ = {isa = PBXBuildFile; productRef = 755C9F5E2BC42B9E00E1F089 /* KeychainAccess */; };
+		755C9F622BC431D900E1F089 /* SnapKit in Frameworks */ = {isa = PBXBuildFile; productRef = 755C9F612BC431D900E1F089 /* SnapKit */; };
+		755C9F642BC431D900E1F089 /* SnapKit-Dynamic in Frameworks */ = {isa = PBXBuildFile; productRef = 755C9F632BC431D900E1F089 /* SnapKit-Dynamic */; };
```
SnapKit-Dynamic тут излишен. Необходимо удалить.

Curshet reviewed (commented):
**1.** Единственный и самый большой размер иконки приложения - удобный, но не всегда хороший вариант, так как иконка масштабируется на устройствах под необходимое разрешение и не всегда это масштабирование будет хорошим. Нужно использовать шаблон, предоставленный Apple. Исправлять не нужно, на будущее.

**2.** Установлены настройки поддерживаемых устройств в таргете приложения, но в тестировании настройки те же.

**3.** Настроенная локализация по умолчанию - английская, не критично, но наверно всё же лучше переключить её на русский язык.

**4.** Необходимо исправить замечание, касающееся SnapKit-Dynamic.

**5.** Излишне длинное/подробное имя MR. Его необходимо писать короче. В предыдущих MR указывал на то, что желательно должно быть в имени MR.

**6.** Желательно нумеровать изменения для большего удобства беглого чтения описания MR. Стоит избегать столь подробного описания действий вплоть до версий библиотек.

**7.** Тестирование UI проводится (в том числе) на Apple iPhone SE (1st gen), не 3rd gen, как это указано в описании MR.

Commit:
Update: testing and localization, remove SnapKit Dynamic

1. The operating system support is installed in the test files - iOS only;
2. The default localization is set to Russia;
3. The project has been tested, including on the iPhone SE device (1st generation)
